vtsclient/vtsclient_grpc.go

```diff
@@ -174,3 +174,19 @@ func (o *GRPC) EnsureConnection() error {
 
 	return nil
 }
+
+func (o *GRPC) GetEARSigningPublicKey(ctx context.Context, in *emptypb.Empty, opts ...grpc.CallOption) (*proto.PublicKey, error) {
+	// var key jwk.Key
+
+	c := o.GetProvisionerClient()
+	if c == nil {
+		return nil, ErrNoClient
+	}
+
+	protoKey, err := c.GetEARSigningPublicKey(ctx, in, opts...)
+	if err != nil {
+		return nil, err
+	}
+
+	return protoKey, nil
```

Comment:
can be simplified to:

```suggestion
	return c.GetEARSigningPublicKey(ctx, in, opts...)
```